src/Elasticsearch/Query.php

```diff
@@ -66,8 +66,12 @@ public function find() : array
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
 
+        $results = curl_exec($ch);
+        if (!$result) {
+            throw new Exception('We have a error with elasticsearch');
```

Comment:
@FredPeal cant we provide something better?


"Error connecting to ElasticSearch Node - because?"

Reply:
@FredPeal lets implements this https://stackoverflow.com/a/3987037/10697357